### ISSUE TITLE
ImportError: dynamic module does not define init function

### ISSUE THREAD

**Issue by rbovill** [pyca/cryptography#3413]
I am using RobotFramework-SSHLibrary on MacOS Sierra.  When I upgraded to the latest version, 2.1.3, I started getting this import error when using the http://robotframework.org/SSHLibrary/latest/SSHLibrary.html#Login%20With%20Public%20Key keyword.  After much digging, I narrowed the problem down to the cryptography package.  If I downgrade to 1.7.1, my tests work just fine.  The full text of the error is this:

ImportError: dynamic module does not define init function (init_constant_time)

**Comment by alex** [comment]
Can you please provide the full traceback, as well as instructions for reproducing, I'm not familiar with RobotFramework.

**Comment by rbovill** [comment]
Unfortunately the text I already provided is all the detail I have.  I tried setting the loglevel, but that didn't provide any more insight.

As for reproducing, here is a robot-framework test suite that shows the error:


`*** Settings ***
Suite Teardown    Close All Connections
Library    SSHLibrary
Resource    Global_Vars.robot

*** Test Cases ***
Login
    Comment    Connect to host.
    Open Connection    host=${Host}    alias=VER    timeout=10s   prompt=${Prompt}
    Comment    Login.
    ${output}=    Login With Public Key    ${UserName}    keyfile=${KeyFile}    password=${PassWord}
`


You'll have to refer to the Robot-Framework and RobotFramework-SSHLibrary documentation for further details.  I have cross-posted this error with the SSHLibrary folks, https://github.com/robotframework/SSHLibrary/issues/171.
